docs/source/models/gbd2017/causes/cause_model_template.rst

```diff
@@ -91,125 +82,193 @@ Disease Description
 
-   #. Remember to adjust the length of the section underline
-      :code:`--------------` to match the length of your text, or you will get
-      errors in the `section structure
-      <https://docutils.sourceforge.io/docs/user/rst/quickref.html#section-structure>`_
-      when Sphinx builds HTML from the :file:`index.rst` file.
+.. admonition:: Todo for template development
 
-   #. Once you complete these steps, delete this :code:`.. important::`
-      `directive <https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#directives>`_
-      from :file:`index.rst`.
+  In `Pull Request 91`_, @James suggested adding the following to this section:
 
-.. admonition:: Todo for template development
+    Point people to yld and cod appendices for their diseases and link to
+    http://ghdx.healthdata.org/gbd-2017/code which can serve as an annoying
+    to read but authoritative source on how the cause was actually modeled.
 
-  - For this section, is it important for the cause name to appear in the
-    section title, as it currently does, or should we call the section
-    something else?
 
-    In `Pull Request 91`_, @James said:
+Cause Hierarchy
++++++++++++++++
 
-      I don't think it needs the cause name. Something like `GBD Modeling
-      Strategy` seems fine.
+Restrictions
+++++++++++++
 
-    Also for this section, @James said:
+The following table describes any restrictions in GBD 2017 on the effects of
+this cause (such as being only fatal or only nonfatal), as well as restrictions
+on the ages and sexes to which the cause applies.
+
+.. list-table:: GBD 2017 Cause Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     -
+     -
+   * - Female only
+     -
+     -
+   * - YLL only
+     -
+     -
+   * - YLD only
+     -
+     -
+   * - YLL age group start
+     -
+     -
+   * - YLL age group end
+     -
+     -
+   * - YLD age group start
+     -
+     -
+   * - YLD age group end
+     -
+     -
+
+
+Vivarium Modeling Strategy
+--------------------------
 
-      Point people to yld and cod appendices for their diseases and link to
-      http://ghdx.healthdata.org/gbd-2017/code which can serve as an annoying
-      to read but authoritative source on how the cause was actually modeled.
+Scope
++++++
 
+.. todo::
 
-Cause Hierarchy
-+++++++++++++++
+  Describe which aspects of the disease this cause model is designed to
+  simulate, and which aspects it is **not** designed to simulate.
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+.. todo::
+
+  Describe the clinical and mathematical assumptions made for this cause model,
+  and the limitations these assumptions impose on the applicability of the
+  model.
+
+.. admonition:: Todo for template development
+
+  What else goes in the `Vivarium Modeling Strategy` section? Should there be
+  any other subsections besides `Scope` and `Assumptions and Limitations`?
```

Comment:
The cause model diagram, data description, and validation criteria are perhaps better suited as subsections of  `Vivarium modeling strategy`